crates/node/src/lib.rs

```diff
@@ -305,29 +300,7 @@ impl Node {
         metadata_writer.update(nodes_config).await?;
 
         if config.common.allow_bootstrap {
-            // only try to insert static configuration if in bootstrap mode
-            let partition_table =
-                Self::fetch_or_insert_initial_configuration(&self.metadata_store_client, &config)
-                    .await?;
-
-            metadata_writer.update(partition_table).await?;
-        } else {
-            // otherwise, just sync the required metadata
-            metadata
-                .sync(MetadataKind::PartitionTable, TargetVersion::Latest)
-                .await?;
-            metadata
-                .sync(MetadataKind::Logs, TargetVersion::Latest)
-                .await?;
-
-            // safety check until we can tolerate missing partition table and logs configuration
-            if metadata.partition_table_version() == Version::INVALID {
-                return Err(Error::SafetyCheck(
-                    format!(
-                        "Missing partition table for cluster '{}'. This indicates that the cluster bootstrap is incomplete. Please re-run with '--allow-bootstrap true'.",
-                        config.common.cluster_name(),
-                    )))?;
-            }
+            // todo write bootstrap state
```

Comment:
what should we expect to do here?

Reply:
This is not fully clear to me yet. It could be the initial default log provider or whether to auto provision partitions if this needs to be agreed upon.